db/migrate/20180905090146_add_theme_data.rb

```diff
@@ -0,0 +1,17 @@
+class AddThemeData < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
this should be `up` I think, as it just runs when applying the migration, not on the rollback